### PR TITLE
Update profile.asciidoc

### PR DESCRIPTION
**Pull request by thinh2**
Fix small typo

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by abdonpijpelink** [comment]
Thanks @thinh2 ! I think we need to fix this sentence, because it is not right:
_time spent while the requests spend in queues_

Maybe you can change it into:
_time that requests spend in queues_

And can you please rebase the PR to target the `main` branch instead of `8.5`?

**Comment by thinh2** [comment]
@abdonpijpelink, thanks for your suggestion. I will update the content and PR also.

**Comment by abdonpijpelink** [comment]
@thinh2 Any progress? I'd be happy to make the changes if you do not have the time.

File: docs/reference/search/profile.asciidoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ The Profile API gives the user insight into how search requests are executed at
 a low level so that the user can understand why certain requests are slow, and
 take steps to improve them. Note that the Profile API,
 <<profile-limitations, amongst other things>>, doesn't measure network latency,
-time spent while the requests spends in queues, or while merging shard
+time spent while the requests spend in queues, or while merging shard
```
Comment (weltenwort) [inline docs/reference/search/profile.asciidoc:20]: I'm not a native speaker, but that sentence still doesn't read quite right. Maybe something like

```suggestion
time spent while the requests wait in queues, or while merging shard
```

or

```suggestion
time the requests spend in queues, or time spent merging shard
```

?

Comment (AndersonQ) [reply docs/reference/search/profile.asciidoc:20]: +1

**Comment by abdonpijpelink** [comment]
@thinh2 Would you mind if I make the requested changes to move this PR forward?

**Review by abdonpijpelink — APPROVED** [review]
LGTM. Thanks @thinh2 !

**Pull request merged**